tests/expected/uninit/multiple-instrumentations.expected

```diff
@@ -1,16 +1,16 @@
-multiple_instrumentations_different_vars.assertion.3\
+multiple_instrumentations_different_vars.assertion.1\
```

Comment:
can you remove these numbers?

Reply:
Will the pattern still match given we use the backslash at the end of the line?